README.adoc

```diff
@@ -274,11 +323,34 @@ The cached system properties contain only the OS name and user name key and valu
 To see that the `system` service is down, point your browser to the http://localhost:9080/system/properties[http://localhost:9080/system/properties^] URL again.
 You see that the service displays a 503 HTTP response code.
 
-When you're done, change the [hotspot=2]`io_openliberty_guides_system_inMaintenance`
-property value back to `false` in the [hotspot file=0]`resources/CustomConfigSource.json` file.
+Point your browser to https://localhost:9443/metrics/application[https://localhost:9443/metrics/application^] again.
+See the following sample outputs for the `@Fallback` annotated method and the fallback method after a fallback occurs:
+
+[role="no_copy"]
+----
+# TYPE application:ft_io_openliberty_guides_inventory_inventory_manager_get_invocations_total counter
+application:ft_io_openliberty_guides_inventory_inventory_manager_get_invocations_total 2
+# TYPE application:ft_io_openliberty_guides_inventory_inventory_manager_get_invocations_failed_total counter
+application:ft_io_openliberty_guides_inventory_inventory_manager_get_invocations_failed_total 0
+# TYPE application:ft_io_openliberty_guides_inventory_inventory_manager_get_fallback_calls_total counter
+application:ft_io_openliberty_guides_inventory_inventory_manager_get_fallback_calls_total 1
+----
 
-include::{common-includes}/mvncompile.adoc[]
+From the output, the `ft_io_openliberty_guides_inventory_inventory_manager_get_invocations_total`
+data indicates that the `@Fallback` annotated method [hotspot=19-21 file=1]`get()` has a total of 2 calls
+including the previous call before turning the `system` service in maintenance.
+The `ft_io_openliberty_guides_inventory_inventory_manager_get_fallback_calls_total` data
+indicates that the [hotspot=23-31 file=1]`fallbackForGet()` method has 1 call.
```

Comment:
Change to:
"From the output, the `ft_io_openliberty_guides_inventory_inventory_manager_get_invocations_total` data indicates that the [hotspot=19-21 file=1]`get()` method was called twice, including the call before the `system` service turned in maintenance. The `ft_io_openliberty_guides_inventory_inventory_manager_get_fallback_calls_total` data indicates that the [hotspot=23-31 file=1]`fallbackForGet()` method was called once."

Reply:
Partially updated